scripts/dts/gen_defines.py

```diff
@@ -560,6 +563,24 @@ def write_gpio_hogs(node: edtlib.Node) -> None:
         for macro, val in macro2val.items():
             out_dt_define(macro, val)
 
+def write_clocks(node: edtlib.Node) -> None:
+    # Write special macros for clock-output-names and clock-state-names properties.
+
+    out_comment("Clock management (clock-output-names, clock-state-names) properties:")
+
+    for prop_name, prop in node.props.items():
+        if prop_name == "clock-output-names":
+            # No reason to use enumerate() here or the next iterator, we use the
+            # index variable in the for loop
+
+            # pylint: disable-next=consider-using-enumerate
+            for i in range(len(prop.val)):
+                out_dt_define(f"{node.z_path_id}_CLOCK_OUTPUT_NAME_{prop.val[i]}_IDX", i)
```

Comment:
```suggestion
            for i, clock_name in prop.val:
                out_dt_define(f"{node.z_path_id}_CLOCK_OUTPUT_NAME_{clock_name}_IDX", i)
```

I think it's saying to do something like this ?

Reply:
we need to use `enumerate` here to get this to work- I've made that change, and pylint no longer complains.

Reply:
sorry, that was a typo, that's what I meant, kind of the worst typo I could have made